plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -508,14 +508,14 @@ public PreparedStatement getPreparedStatement(Connection connection, String sql)
             }
 
             case Types.CHAR:
-                return Optional.of(charColumnMapping(typeHandle.getRequiredColumnSize()));
+                return Optional.of(charColumnMapping(typeHandle.getRequiredColumnSize(), isCollatable(jdbcTypeName)));
```

Comment:
just `true`? 

ie what are PostgreSQL data types for which we would be here & `isCollatable`  would return false?

Reply:
Ah, `CHAR` types can be always `true`. I dropped the `withCollate` parameter from `charColumnMapping()`.